File: Tarea1Unidad3Python/Electrodomesticos.py
from Producto import Producto
from Producto import Enum, tipoProducto

class Electrodomesticos(Producto):
    def __init__(self, nombre, precio, stock, fecha_importacion, marca):
        super().__init__(nombre, tipoProducto.ELECTRODOMESTICOS, precio, stock, fecha_importacion)
        self.marca = marca

    def get_marca(self):
        return self.marca

    def get_datos(self):
        return f"{super().get_datos()}    Marca: {self.get_marca()}"

File: Tarea1Unidad3Java/src/Productos/Electrodomesticos.java
package Productos;

import Productos.helpers.tipoProducto;

public class Electrodomesticos extends Producto {
    private  String marca;
    public Electrodomesticos(String nombre, double precio, int stock, String fechaImportacion, String numSerie, String marca) {
        super(nombre, tipoProducto.ELECTRODOMESTICOS, precio, stock, fechaImportacion, numSerie);
        this.marca=marca;
    }

    public String getMarca() {
        return marca;
    }

    @Override
    public String getDatos(){
        return String.format("%s    Marca: %s", super.getDatos(), getMarca());
    }
}
